ocs_ci/deployment/vmware.py

```diff
@@ -1632,16 +1632,21 @@ def destroy_cluster(self, log_level="DEBUG"):
             config.ENV_DATA["vsphere_user"],
             config.ENV_DATA["vsphere_password"],
         )
-        all_vms = vsphere.get_vms_by_string(config.ENV_DATA["cluster_name"])
-        vsphere.stop_vms(all_vms)
+        try:
+            all_vms = vsphere.get_vms_by_string(config.ENV_DATA["cluster_name"])
+            vsphere.stop_vms(all_vms)
 
-        for vm in all_vms:
-            try:
-                vsphere.remove_disks_with_main_disk(vm)
-            except Exception as e:
-                logger.error(
-                    f"Removing disks for {vm.name} in destroy fail with the error {e}"
-                )
+            for vm in all_vms:
+                try:
+                    vsphere.remove_disks_with_main_disk(vm)
+                except Exception as e:
+                    logger.error(
+                        f"Removing disks for {vm.name} in destroy fail with the error {e}"
+                    )
+        except Exception as e:
```

Comment:
Move to cover just get_vms_by_string and vsphere.stop_vms as later block is already covered in try except block